README.md

```diff
@@ -22,12 +22,12 @@ import (
 
 func TestSubscription(t *testing.T) {
     server := cerulean.New("c27e7a81-b684-4fce-91d8-fed9e9bb534a")
-	client := subscriptions.NewClientWithBaseURI(server.GetBaseClientURI())
-	client.Authorizer = autorest.NullAuthorizer{}
+    client := subscriptions.NewClientWithBaseURI(server.GetBaseClientURI())
+    client.Authorizer = autorest.NullAuthorizer{}
 
-	resultPage, err := client.List(context.TODO())
-	if err != nil {
-		panic(err)
+    resultPage, err := client.List(context.TODO())
+    if err != nil {
+	    panic(err)
```

Comment:
good catch

Reply:
Mostly noticed because I added error handling to the snippet during my iterating session and then ripped it out. Didn't notice until I'd ripped. 😛